src/test/incremental/issue-101518.rs

```diff
@@ -0,0 +1,31 @@
+// revisions: cfail1 cfail2
```

Comment:
It seems like these aren't actually used?

Reply:
Incremental test seem to require revisions. But I suppose we only need one.